ethereum/consts.go

```diff
@@ -0,0 +1,15 @@
+package contracts
+
+// Gas prices for our operations. Most of these are set by the highest value we
+// ever see in a test, so you would need to adjust upwards a little to use as a
+// gas limit. We use these values to estimate minimum required balances.
```

Comment:
I think the values below have stabilized, but if a test fails when working on a future PR, just increment the constant to the new recorded high.